html/manifest/scope.json

```diff
@@ -7,7 +7,7 @@
           "spec_url": "https://w3c.github.io/manifest/#scope-member",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": "53"
             },
             "chrome_android": {
               "version_added": null
```

Comment:
Is there a reason this was set only for Chrome and not Chrome for Android? This is different from the others in this PR.

Reply:
I think I just forgot to add it in

Reply:
91851d8